docs/readme.md

```diff
@@ -0,0 +1,68 @@
+# BENGALURU HOUSE PRICE PREDICTION
```

Comment:
Can you rename the readme file since we have an existing readme.md file?
Maybe a name like team_tasks.md or some else that is intuitive